.flake8

```diff
@@ -3,3 +3,11 @@ max-line-length = 100
 enable-extensions = L101,L102
 ignore = E402
 exclude=*.egg/*
+
+# Configuration for flake8-copyright extension
+copyright-check = True
```

Comment:
I thought copyright-check was already enabled. This is why i got licensing pylint error.

Reply:
This needs to be enabled explicitly to have the copyright check.